sentry/src/main/java/io/sentry/ITransaction.java

```diff
@@ -77,4 +78,11 @@ public interface ITransaction extends ISpan {
 
   /** Schedules when transaction should be automatically finished. */
   void scheduleFinish();
+
+  @ApiStatus.Internal
```

Comment:
Dunno if it makes sense to have them public?

Reply:
Do you mean not marking them with `@ApiStatus.Internal` or not having them on this interface?

For now this should be something only we use. If we find out otherwise we can always easily remove it. Other way around is not easily possible.

Reply:
I meant not marking the internal, but what you said makes sense, let's keep it 👍